polyfills/IntersectionObserver/config.toml

```
@@ -26,9 +26,9 @@ firefox = "< 55"
 firefox_mob = "< 55"
 ie = "*"
 ie_mob = "*"
-ios_saf = "*"
+ios_saf = "< 12.2"
```

```suggestion
ios_saf = "*"
```

perhaps I could annotate this in the `notes = [ ]` section? curious if you have a ref, or if this perhaps because of the `options.root parameter can be a Document: NO` constraint?